apps/demo/src/app/pages/notification/examples/02-create-notification/index.ts

```diff
@@ -0,0 +1,34 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {NotificationService} from '@ng-web-apis/notification';
+import {PermissionsService} from '@ng-web-apis/permissions';
+import {filter, switchMap} from 'rxjs/operators';
+
+@Component({
+    selector: 'notification-page-example-2',
+    templateUrl: './index.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NotificationPageExample2 {
+    readonly notificationPermissionState$ =
+        this.permissionsService.state('notifications');
+
+    constructor(
+        private readonly notificationService: NotificationService,
+        private readonly permissionsService: PermissionsService,
+    ) {}
+
+    sendNotification(): void {
+        this.notificationService
+            .requestPermission()
+            .pipe(
+                filter(permission => permission === 'granted'),
```

Comment:
We probably need such filter in the permissions package

Reply:
I've introduced new rxjs-operator:

```ts
observable$..pipe(
    filterPermission('granted'),
)
```

✅